kdump-utils.spec

```diff
@@ -75,21 +67,14 @@ servicelog_notify --remove --command=/usr/lib/kdump/kdump-migrate-action.sh >/de
 %systemd_preun kdump.service
 
 %posttrans
-# don't try to systemctl preset the kdump service for old kexec-tools
-if [[ -f %{kexec_tools_no_preset} ]]; then
-  # this if branch can be removed in F41 when we assume no users will use the old kexec-tools
-  rm %{kexec_tools_no_preset}
-else
-  # Initial installation
-  %systemd_post kdump.service
```

Comment:
Thanks for raising the question! This is because %posttrans runs after all packages' %post
scriptlets have been executed. I've updated the commit message to include this info and rephrase the msg to make it more comprehensive. If you still have any doubt, feel free to let me know.

Reply:
I expect an explicit sync instruction. But the implicit is good enough too.

Thanks

Reply:
Thanks for confirming it! I don't think dnf or rpm currently provides a way to control the order of different packages' RPM scriptlets:)